### DISCUSSION TITLE
Keyboard jogging A Axis

### DISCUSSION THREAD

**Discussion by Switch-Witch** [terjeio/ioSender#413] [Q&A]
Hi, does somebody know wich keys to use to jog the 4th axis (a) with the keyboard?

best

Kai

**Comment by terjeio** [comment]
Continuous jogging: \<Home\> and \<End\> buttons, can use with \<ctrl\> and \<shift\> modifiers.
Triggering _Jog_ panel buttons: \<ctrl\>+\<shift\>+\<U\> and \<ctrl\>+\<shift\>+\<N\>. Distance and feed rate from the _Jog_ panel selections.

**Comment by Switch-Witch** [comment]
Hi Terjeo, thx (again) for your answer! Does this mean that <hone> and <End> are the buttons to rotate clockwise/counterclockwise?

**Reply by terjeio** [reply]
Yes, just try it.
